Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -108,6 +115,11 @@ private void initializeMotor() {
         motor.configSelectedFeedbackSensor(FeedbackDevice.CTRE_MagEncoder_Relative, 0, 0);
         motor.setSensorPhase(true);
 
+        motor.configPeakCurrentLimit((int) elevatorPeakCurrentLimit.get(), 0);
```

Comment:
Property changes will never be propagated to the Talon. Do we want to do something like @JohnGilb did for the drive?